### ISSUE TITLE
Bad validation loss and wrong predictions

### ISSUE THREAD

**Issue by Hu9o73** [Hu9o73/PictaScript#1]
The validation loss tends to go up as the train loss goes down, most certainly indicating overfitting.

![Image](https://github.com/user-attachments/assets/348b3d97-df3c-4812-92fb-d42c00d2e1ff)

This later yields unexact results on images that aren't part of the dataset:

![Image](https://github.com/user-attachments/assets/8eb2d32a-3929-46b3-baac-eb1751ab396d)

![Image](https://github.com/user-attachments/assets/3eb24b78-5783-42da-bc52-7cd04349d33c)
